docs/sources/tempo/metrics-from-traces/span-metrics/span-metrics-metrics-generator.md

```diff
@@ -83,7 +83,7 @@ By default, the metrics processor adds the following labels to each metric: `ser
   - `STATUS_CODE_ERROR` - The span operation completed with an error
 - `status_message` (optionally enabled) - The message that details the reason for the `status_code` label
 - `job` - The name of the job, a combination of namespace and service; only added if `metrics_generator.processor.span_metrics.enable_target_info: true`
-- `instance` - The instance ID; only added if `metrics_generator.processor.span_metrics.enable_target_info: true`
+- `instance` - The instance ID; only added if `metrics_generator.processor.span_metrics.enable_target_info: true` and can be dropped by setting `metrics_generator.processor.span_metrics.drop_instance_label: true
```

Comment:
@rlankfo isn't instance semi required for `target_info` to work correctly?

Reply:
IE when we go to set the value on the Gauge your likely to override/not override appropriately in SetForTargetInfo

Reply:
If there is not a job or instance then the metric will never get recorded, which feels like different behavior than reducing cardinality. The change in behavior would be if there is no job label and drop instance is true then the metric will not be recorded, whereas previously it would be. Though granted that should be pretty rare that a `service.name` is not declared.